packages/contracts/contracts/L1/rollup/OVM_CanonicalTransactionChain.sol

```diff
@@ -37,6 +37,8 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
     uint256 constant public MIN_ROLLUP_TX_GAS = 100000;
     uint256 constant public MAX_ROLLUP_TX_SIZE = 50000;
     uint256 constant public L2_GAS_DISCOUNT_DIVISOR = 32;
+    uint256 constant public ENQUEUE_GAS_CONSUMED = 60000;
```

Comment:
nit: `ENQUEUE_GAS_COST` reflects that this is how much enqueue costs regardless of whether we ran it.

Reply:
What you mean by
> regardless of whether we ran it
?

Reply:
Also, are you suggesting I change the var name to `ENQUEUE_GAS_COST`?

Reply:
Sorry, yes, that's a var name suggestion.